src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -1044,10 +1000,19 @@ public void setPortNumber(int userPort) {
 	 *
 	 * @return the currently registered user's chat ID.
 	 */
-	public String getUserName() {
+	public static String getUserName() {
```

Comment:
Why was this method made static? It does not make sense that all instances of the class and even no instance of the class would have the same username associated.